### ISSUE TITLE
[FEATURE REQUEST]  suggester to value syntax

### ISSUE THREAD

**Issue by Incblob** [chhoumann/quickadd#148]
# Limitation:
while you can add suggestions using {{VALUE: option, option2}}, these suggestions cannot be reused. 

You can also do {{VALUE:<var1>,<var2>}}, which gives you the option between two existing variables, but then you either have to define the variables in a macro or define the variables in a prompt again, without a suggester. 

# Suggested solution
have a syntax like {{val_name:option1,option2}} or {{<value_name>: opt1, opt2}} which sets the variable val_name to that option, enabling reuse. 
Basically a way to set a value using a suggester and then reuse it.

# context
I'm trying to have dynamic note titles which contain the note category, for example:
FileName: {{category: cat1, cat2, cat3}} - {{title}}
and then automatically add the category as a tag in the note:
tags: #{{category}}
